website/content/en/status/report-2024-01-2024-03/cloud-init.adoc

```diff
@@ -0,0 +1,34 @@
+=== FreeBSD as a Tier 1 cloud-init Platform
+
+Links: +
+link:https://cloud-init.io/[cloud-init Website] URL: link:https://cloud-init.io/[] +
+link:https://cloudinit.readthedocs.io/en/latest/[cloud-init Documentation] URL: link:https://cloudinit.readthedocs.io/en/latest/[] +
+
+Contact: Mina Galić <freebsd@igalic.co>
+
+cloud-init is the standard way of provisioning servers in the cloud.
+Over the past year and a half, thanks to this FreeBSD support has steadily improved.
+This year, we took a pivot and decided to explicitly focus on making improvements in FreeBSD itself, that will aid the cloud-init team to test future changes to FreeBSD code-paths themselves.
+To achieve this goal, we need to make FreeBSD run in LXD (and Incus), which it already can, but not under control of ``lxd-agent`` (or ``incus-agent``).
+
+Here are some improvements from the recent weeks:
+
+- I have written a small https://codeberg.org/meena/test-cloud-init[testing-framework] (in sh, and I'm slowly porting it to OpenTofu/Terraform), which installs the latest version of package:net/cloud-init-devel[] or package:net/cloud-init[] and runs a couple of standard cloud-init tests.
+- To do this, I have created a https://pkg.igalic.co/[dedicated public repository] which contains the latest versions of package:net/cloud-init-devel[] and package:net/cloud-init[] for FreeBSD 13 and 14 on amd64 and aarch64
+- I have ported Linux' vsock testing framework to https://codeberg.org/meena/vsock-tests[FreeBSD]
+- Created the baseline of a https://codeberg.org/meena/freebsd-src/src/branch/vsock/sys/dev/virtio/socket[VirtIO Socket driver], based on the HyperV Socket driver.
```

Comment:
s/Created/I created/ maybe, for consistency with other bullet points.

Also, can you clarify what you mean by "baseline" here?

Reply:
An alternative which would work in case you are not the author of this baseline is "A baseline has been created".